pkg/operators/common/kubeinventorycache.go

```diff
@@ -189,13 +197,15 @@ func (cache *inventoryCache) OnAdd(obj any, _ bool) {
 			return
 		}
 		cache.pods.Add(key, o)
+		cache.podsByIp.Add(o.Status.PodIP, o)
```

Comment:
What is the value of PodIP when the pod has `hostNetwork=true`?

During initialization, before the CNI assigns an IP to the pod, is `o.Status.PodIP` initialized to `""`?

Would it causes problems if the pod (or several pods) has `o.Status.PodIP=""`?